website/docs/r/privatelink_endpoint.html.markdown

```diff
@@ -0,0 +1,124 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: private_endpoint"
+sidebar_current: "docs-mongodbatlas-resource-private_endpoint"
+description: |-
+    Provides a Private Endpoint resource.
+---
+
+# mongodbatlas_privatelink_endpoint
+
+`mongodbatlas_privatelink_endpoint` provides a Private Endpoint resource. This represents a Private Endpoint Service that can be created in an Atlas project.
+
+~> **IMPORTANT:**You must have one of the following roles to successfully handle the resource:
+  * Organization Owner
+  * Project Owner
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+-> **NOTE:** A network container is created for a private endpoint to reside in if one does not yet exist in the project.  
+
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_privatelink_endpoint" "test" {
+  project_id    = "<PROJECT-ID>"
+  provider_name = "AWS/AZURE"
+  region        = "us-east-1"
+}
+```
+
+## Argument Reference
+
+* `project_id` - Required 	Unique identifier for the project.
+* `providerName` - (Required) Name of the cloud provider for which you want to create the private endpoint service. Atlas accepts `AWS` or `AZURE`.
+* `region` - (Required) Cloud provider region in which you want to create the private endpoint connection.
+Accepted values are:
```

Comment:
We have had a hard time keeping these updated so please just link to the api doc for these, for example:
Updated accepted values can found in the main API documentation: [MongoDB API Private Endpoint Service](https://docs.atlas.mongodb.com/reference/api/private-endpoints-service-create-one//)